source/backups.rst

```diff
@@ -3,14 +3,17 @@ Providing Backups
 
 Percona XtraDB Cluster Operator allows doing cluster backup in two ways.
 *Scheduled backups* are configured in the
-`deploy/cr.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml>`__
+`deploy/cr.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml>`_
 file to be executed automatically in proper time. *On-demand backups*
 can be done manually at any moment.
 
 Backup images are usually stored on `Amazon S3 or S3-compatible
-storage <https://en.wikipedia.org/wiki/Amazon_S3#S3_API_and_competing_services>`__
-(storing backups on private storage is also possible, but they are
-described separately).
+storage <https://en.wikipedia.org/wiki/Amazon_S3#S3_API_and_competing_services>`_,
+but storing backups on a private volume is also possible.
```

Comment:
```suggestion
but storing backups on Persistent Volumes is also possible.
```

please, s/private volume/Persistent Volumes/ everywhere.
we use one volume per one backup so the plural form is better - Volumes